instance/migrations/0049_appserver_refactor2.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-05-04 04:29
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django_extensions.db.fields
+import instance.models.instance
+import instance.models.mixins.utilities
+import instance.models.utils
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contenttypes', '0002_remove_content_type_name'),
+        ('instance', '0048_appserver_refactor1'),
+        ('pr_watch', '0001_initial'),
+    ]
+
+    def data_forward(apps, schema_editor):
+        """
+        Move existing data from SingleVMOpenEdXInstance to OpenEdXInstance + OpenEdXAppServer
+        """
+        # Unchanged models:
+        ContentType = apps.get_model('contenttypes', 'ContentType')
```

Comment:
I didn't realize it would work in uppercase, actually. I will change them all to camel case.